scripts/partition_manager/partition_manager.rst

```diff
@@ -0,0 +1,408 @@
+
+For this purpose, Partition Manager provides static configuration to define static partitions.
+The area used by the static partitions is called the *static area*.
+The static area comes in addition to the *dynamic area*, which consists of the ``app`` partition and all memory adjacent to the ``app`` partition that is not occupied by a static partition.
+Note that there is only one dynamic area.
+When Partition Manager is executed, it operates only on the dynamic area, assuming that all other memory is reserved.
+
+Within the dynamic area, you can define new partitions or configure existing partitions even if you are using static partitions.
+The dynamic area is resized as required when updating the static configuration.
+
+.. _ug_pm_static_providing:
+
+Configuring static partitions
+=============================
+Static partitions are defined through a YAML-formatted configuration file in the root application's source directory.
+This file is similar to the regular :file:`pm.yml` configuration files, except that it also defines the start address for all partitions.
+
+If the build system discovers a file named :file:`pm_static.yml` in an application's source directory, it automatically provides it to the Partition Manager script as static configuration.
+
+The current partition configuration for a build can be found in :file:`${BUILD_DIR}/partitions.yml`.
+To apply the current configuration as a static configuration, copy this file to :file:`${APPLICATION_SOURCE_DIR}/pm_static.yml`.
+
+You can add or remove partitions as described in the following sections.
+
+.. note::
+  If the static configuration contains an entry for the `app` partition, this entry is ignored.
+
+.. _ug_pm_static_remove:
+
+Removing a static partition
+---------------------------
+To remove a static partition, delete its entry in :file:`pm_static.yml`.
+
+Only partitions adjacent to the ``app`` partition or other removed partitions can be removed.
+
+.. _ug_pm_static_add_dynamic:
+
+Adding a dynamic partition
+--------------------------
+New dynamic partitions that are listed in a :file:`pm.yml` file are automatically added.
+However, if a partition is defined both as static partition and as dynamic partition, the dynamic definition is ignored.
+
+.. note::
+   When resolving the relative placement of dynamic partitions, any placement properties referencing static partitions are ignored.
+
+.. _ug_pm_static_add:
+
+Adding a static partition
+-------------------------
+To add a static partition, add an entry for it in :file:`pm_static.yml`.
+This entry must define the properties ``address``, ``size``, and - if applicable - ``span``.
+
+.. code-block:: yaml
+   :caption: Example of static configuration of a partition with span
+
+   partition_name:
+      address: 0xab00
+      size: 0x1000
+      span: [example]  # Only if this partition had the span property set originally.
+
+.. note::
+  Sub-images with the build strategy *partition_name*\ _BUILD_STRATEGY_SKIP_BUILD or
```

Comment:
"with the build strategy" - sub-images that are built with these options set? Or should sub-images provide different processing depending on these options?

Reply:
`sub-images that are built with these options set` will also be correct. So choose that if it is better.